### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by kesla**
Add readme

R = @iefserge 
T = patch

### CODE REVIEW

**Comment by kesla** [comment]
@iefserge 

Hey, here's a readme. There's definitely pieces missing here - but this is a start.

Please look through the api / format and let me know if there's anything that look confusing to you, or in some other way unclear!

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "children": [{
+    "type": "text",
+    "content": "Hello, ",
+    "href": null,
+    "italic": false,
+    "bold": false
+  }, {
+    "type": "text",
+    "content": "mic.com",
+    "href": "http://wwww.mic.com",
```
Comment (iefserge) [inline readme.md]: www

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  "caption": [{
+    "type": "text",
+    "content": "Here's a video from ",
+    "href": null,
+    "italic": false,
+    "bold": false
+  }, {
+    "type": "text",
+    "content": "mic.com",
+    "href": "http://wwww.mic.com",
```
Comment (iefserge) [inline readme.md]: www

**Comment by iefserge** [comment]
some comments, otherwise :+1:

**Pull request merged**